src/server/pkg/testpachd/mock_env.go

```diff
@@ -0,0 +1,85 @@
+package testpachd
+
+import (
+	"context"
+
+	"golang.org/x/sync/errgroup"
+
+	"github.com/pachyderm/pachyderm/src/client"
+	foo "github.com/pachyderm/pachyderm/src/server/pkg/testetcd"
```

Comment:
`foo`?

Reply:
ugh - I ran into build problems (probably from the wrong package name that you spotted above), this was the workaround =(

fixed